cli/src/semgrep/app/scans.py

```diff
@@ -154,10 +154,9 @@ def fetch_and_init_scan_config(self, meta: Dict[str, Any]) -> None:
         if self.dry_run:
             app_get_config_url = f"{state.env.semgrep_url}/{DEFAULT_SEMGREP_APP_CONFIG_URL}?{self._scan_params}"
         else:
-            app_get_config_url = f"{state.env.semgrep_url}/{DEFAULT_SEMGREP_APP_CONFIG_URL}?{self._scan_params}"
```

Comment:
It concerns me that there are no params in the new endpoint. How does it determine if it's a forced sca only scan?

Reply:
This should be sent now in the `start_scan` function, which is now called before this. However, looking in the GitMeta to_dict method, I don't actually see that specific argument anymore, so we should definitely verify that all the pieces the app expects are in fact being sent by the CLI, good call!

Reply:
Oh, actually, the same metadata dict that is passed into this function is sent in its entirety to the start_scan endpoint, so it should be good 👍